EIPS/eip-1.md

```diff
@@ -67,54 +87,112 @@ These calls generally result in a "rough consensus" around what EIPs should be i
 
 *In short, your role as the champion is to write the EIP using the style and format described below, shepherd the discussions in the appropriate forums, and build community consensus around the idea.*
 
+
 ### EIP Process
 
 The following is the standardization process for all EIPs in all tracks:
 
 ![EIP Status Diagram](../assets/eip-1/EIP-process-update.jpg)
 
-**Idea** - An idea that is pre-draft. This is not tracked within the EIP Repository.
+#### Idea
 
-**Draft** - The first formally tracked stage of an EIP in development. An EIP is merged by an EIP Editor into the EIP repository when properly formatted.
+An idea that is pre-draft. This is not tracked within the EIP Repository.
 
-**Review** - An EIP Author marks an EIP as ready for and requesting Peer Review.
+#### Draft
 
-**Last Call** - This is the final review window for an EIP before moving to `Final`. An EIP editor will assign `Last Call` status and set a review end date (`last-call-deadline`), typically 14 days later.
+The first formally tracked stage of an EIP in development. An EIP is merged by an EIP Editor into the EIP repository when it meets some initial requirements: It should be technically sound, the Title and Preamble must be correct, and all included Sections should be readable.  Problems with language like spelling, grammar, markup, etc. are only considered warnings.
```

Comment:
I would drop the "technically sound" requirement, and expand a bit:

```suggestion
Draft is the first formally tracked stage of an EIP in development. An EIP is merged by an EIP Editor into the EIP repository when it meets some initial requirements: the title and preamble must be complete and descriptive, and required sections must be present and readable (but not necessarily complete.)  Problems with language like spelling, grammar, etc. are only considered warnings.

The specification itself does not need to be completely fleshed out, but should contain enough content to give a good idea of the intent of the proposal. Importantly, the proposal must be a good fit for the EIP process, as defined by the EIP Editors.
```

Reply:
I didn't change the "technically sound" part, and am inclined to keep it.  Otherwise I'm fine with your suggesting.

Reply:
Also, I'm not sure what "a good fit for the EIP process" means.  I wrote "must actually be a proposal to improve Ethereum", but I'm not sure if that's better.

Reply:
I'd like to include the rule somewhere that editors can reject things that have little business being standards.

For example, we might reject:

 * `EIP-XXXX: Solidity Style Guide` because it isn't a technical specification, 
 * `EIP-XXXX: An implementation of secp256r1` because it doesn't define how two or more parties interoperate, or
 * `EIP-XXXX: A comprehensive table of precompile addresses` because it can never go to final.

I'm not sure how to write that concisely without unduly limiting editor discretion though.

Reply:
^ I tend to agree with Sam's point, that:
- never being finalizable 
- not improving interop at any level of the EVM stack (incl wallet-to-wallet, for ex)
and
- not improving the functioning of clients
should all be valid reasons for saying no.  i defer to the group where/how best to insert these restrictions, but that certainly seems a good starting point for acceptance criteria.

Sidenote: by both *my*  version of Sam's three rules and his, an INFORMATIONAL EIP defining EVM "compatibility" could be debatably in or out of scope.  That implies to me that the scope of "interop" should perhaps be more explicit, so it's less subjective whether defining EVM (at least for 2023) is a good EIP document or not :D

Reply:
I agree with where this is going, but would rather not resolve it just yet.